qiskit/transpiler/exceptions.py

```diff
@@ -15,17 +15,20 @@
 """
 Exception for errors raised by the transpiler.
 """
+from deprecated import deprecated
 from qiskit.exceptions import QiskitError
 
 
-class TranspilerError(QiskitError):
-    """Exceptions raised during transpilation"""
-
-
+@deprecated(version="0.10.0",
```

Comment:
0.10.0 has been released since this was first pushed, can you bump this to be `0.11.0`.

Reply:
Done